.github/workflows/format-and-lint.yml

```diff
@@ -0,0 +1,26 @@
+name: Format and Lint
+on: [push]
+jobs:
+  format-and-lint:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+
+      - name: Use Node.js 14.x
+        uses: actions/setup-node@v1
+        with:
+          version: 14.x
+
+      - name: Install deps and build (with cache)
+        uses: bahmutov/npm-install@v1
+
+      - name: Format
+        run: yarn format-write
```

Comment:
This might cause the CI pipeline to fail because of a single line of code misformatted. `--write` just formats the files in place which is what is happening in the current setup. in my opinion, a pre-commit hook that checks the formatting is more useful.

Reply:
It should fail if a line is not formatted correctly. This is what happens in the current setup (`yarn lint`). If you write to it in CI the format error will still be in the code. It'll only be formatted in the CI files, not the actual committed ones. A pre-commit hook that writes would be useful for this.

Reply:
Ops, I thought `yarn lint-fix` is running in CI.
So do you agree with a husky setup?

Reply:
I think that'd be fine but might want to wait for Alex on that.

Reply:
I don't like husky - it slows down my flow. I trust my editor to format correctly